recipes/mimsi/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.4.2" %}
+
+
+package:
+  name: mimsi
+  version: {{ version }}
+
+source:
+  url: https://github.com/mskcc/mimsi/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b5d8aa6012c5b452e100d9a7940cb262b491976e0f83fd90c9b56b94114e629c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir "
+
+requirements:
+  host:
+    - python >=2.7
+    - python <3.9
+    - setuptools
+    - pip
+
+  run:
+    - python >=2.7
+    - python <3.9
+    - future
+    - nose >=1.3.7
+    - pytorch
+    - torchvision
+    - numpy ==1.14.6 # [py==35]
+    - numpy # [py>=36]
+    - numpy # [py==27]
```

Comment:
What are you trying to achieve here?

Reply:
I was having version conflicts while doing prior installs between numpy, pytorch, sci-kit and pandas on older versions of python with pip, and I carried over the versions into the conda recipe. I've removed those pinned dependencies and set the supported versions of python to 3.6, 3.7 and 3.8. I think that should simplify things considerably.